routes/web.php

```diff
@@ -21,9 +21,16 @@
 $router->get('campaigns/{id}/inbox', 'Legacy\Web\AshesCampaignsController@showInbox');
 $router->get('campaigns/{id}', 'Legacy\Web\AshesCampaignsController@showCampaign')->name('campaigns.show');
 // Create, update, delete campaigns via Rogue.
-$router->post('campaigns', 'Legacy\Web\CampaignsController@store');
-$router->patch('campaigns/{campaign}', 'Legacy\Web\CampaignsController@update');
-$router->delete('campaigns/{campaign}', 'Legacy\Web\CampaignsController@destroy');
+// @TODO: rename these routes (to just /campaigns) when we get off Ashes.
+$router->resource('campaign-ids', 'Legacy\Web\CampaignsController', [
+    'names' => [
```

Comment:
You shouldn't need to manually set names for each of these routes, since the `resource` helper should assign these same defaults for you. This can just be:

```php
$router->resource('campaign-ids', 'Legacy\Web\CampaignsController');
```

Reply:
oh duh - thanks @DFurnes ! It wasn't working when I did that at first and added the names array because I forgot to change the references to the names elsewhere to what they're actually supposed to be! This is updated :)